File: src/commands/image.rs
use image;

pub fn resize_image(file_path: &str, output_path: &str, width: u32, height: u32) {
    let img = image::open(file_path).unwrap();
    let resized_img = img.resize_exact(width, height, image::imageops::FilterType::Nearest);
    resized_img.save(output_path).unwrap();
}

File: src/dkdc/image.py
from PIL import Image


def resize_image(input_path: str, output_path: str, height: int, width: int) -> None:
    """
    Resize an image.
    """
    with Image.open(input_path) as img:
        resized_img = img.resize((height, width))
        resized_img.save(output_path)
